website/docs/r/redis_cache.html.markdown

```diff
@@ -73,6 +73,14 @@ The following arguments are supported:
 
 ~> **Note:** Configuring the number of replicas per master is only available when using the Premium SKU and cannot be used in conjunction with shards.
 
+* `replicas_per_primary` - (Optional) Amount of replicas to create per primary for this Redis Cache. If both `replicas_per_primary` and `replicas_per_master` are set, they need to be equal.
+
+* `redis_version` - (Optional) Redis version. Only major version needed. Valid values: `4`, `6`. 
```

Comment:
If it's optional, what is the default value ?

Reply:
Its computed so its derived from the API (and can change depending on what azure defaults too)

Reply:
@BzSpi For current API, the default value will be 4.